bin/p2-bootstrap/bootstrap.go

```diff
@@ -39,7 +40,7 @@ func main() {
 		if err != nil {
 			log.Fatalf("Could not get consul manifest: %s", err)
 		}
-		consulPod = pods.NewPod(consulManifest.ID(), pods.PodPath(*podRoot, consulManifest.ID()))
+		consulPod = pods.NewPod(string(consulManifest.ID()), pods.PodPath(*podRoot, consulManifest.ID()))
```

Comment:
That makes sense, however I'd argue that it's weird to cast back to `string` if you do have a manifest object though. I'd still be in favor of making `NewPod` take a `types.PodID`